front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -183,7 +197,12 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
           buttonStyle="secondary"
           onClick={(e: MouseEvent) => {
             e.preventDefault();
-            scrollTo('project-ideas');
+            scrollTo({
```

Comment:
Minor: It looks like the `pathname`, `projectSlug` and `currentPhase` are always passed in when using `scrollTo` in this file. We could potentially reuse 🤔

Reply:
I've changed it, but I'm actually not a fan of making changes like this after most QA testing has happened. Very easy to introduce bugs.